java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -144,11 +144,12 @@ public static MerkleTreeLeaf parseMerkleTreeLeaf(InputStream in) {
     }
 
     int leafType = (int) readNumber(in, 1);
-    if (leafType != Ct.MerkleLeafType.TIMESTAMPED_ENTRY_VALUE) {
+    // timestamped_entry == 0
+    if (leafType != 0) {
```

Comment:
public static final int A_CONSTANT = 42;

or possibly not public depending on how you use it.

Martin

On 10 June 2016 at 14:14, Pierre Phaneuf <notifications@github.com> wrote:

> In
> java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java
> <https://github.com/google/certificate-transparency/pull/1246#discussion_r66610364>
> :
>
> > @@ -144,11 +144,12 @@ public static MerkleTreeLeaf parseMerkleTreeLeaf(InputStream in) {
> >      }
> >
> >      int leafType = (int) readNumber(in, 1);
> > -    if (leafType != Ct.MerkleLeafType.TIMESTAMPED_ENTRY_VALUE) {
> > +    // timestamped_entry == 0
> > +    if (leafType != 0) {
>
> I'm going to have to look at how you do a constant in Java, then? 😃
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/certificate-transparency/pull/1246/files/53c4ec90f94a596e9c1655577343386efac77e5d#r66610364>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMv2T4g_Ni4Egf91eMVd0kSpcfmiea7Mks5qKWMdgaJpZM4IyLWD>
> .
>